integration/inspektor-gadget/integration_test.go

```diff
@@ -51,7 +51,7 @@ var (
 	dnsTesterImage = flag.String("dnstester-image", "ghcr.io/inspektor-gadget/dnstester:latest", "dnstester container image")
 
 	doNotDeployIG  = flag.Bool("no-deploy-ig", false, "don't deploy Inspektor Gadget")
-	doNotDeploySPO = flag.Bool("no-deploy-spo", false, "don't deploy the Security Profiles Operator (SPO)")
+	doNotDeploySPO = flag.Bool("no-deploy-spo", true, "don't deploy the Security Profiles Operator (SPO)")
```

Comment:
I agree to keep the code because it will be helpful to test the advise seccomp-profile gadget, see https://github.com/inspektor-gadget/inspektor-gadget/issues/975.